crates/router/src/core/payment_methods/cards.rs

```diff
@@ -4823,7 +4846,34 @@ pub async fn get_bank_account_connector_details(
                     connector_details: connector_details.clone(),
                 };
 
-                Ok(Some(token_data))
+                let token_data = PaymentTokenData::AuthBankDebit(token_data);
+                let retval = PaymentMethodListContext {
```

Comment:
```suggestion
                let retval = PaymentMethodListContext {
```
nit: please use better naming convention